R/layers.R

```diff
@@ -1086,6 +1086,8 @@ clearShapes = function(map) {
 
 #' @param geojson a GeoJSON list, or character vector of length 1
 #' @describeIn map-layers Add GeoJSON layers to the map
+#' @param popupProperty The property to use for popup content
+#' @param labelProperty The property to use for the label.
```

Comment:
Can you document that these can either be the names of the properties, or JavaScript functions? With an example?